card-set/src/card-component.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import './Card.css';
+
+const UserCard = (props) => {
+  const [avatar, setAvatar] = React.useState('https://media.tenor.com/images/e549f9798674301c5af2c91581194091/tenor.gif')
+  const [firstName, setFirstName] = React.useState('Loading...')
+  const [lastName, setLastName] = React.useState('Loading...')
+  const [email, setEmail] = React.useState('Loading...')
+  
+  React.useEffect(() => {
+    fetch(`https://reqres.in/api/users/${props.userId}`)
```

Comment:
One thing that can help keep your components a little bit shorter is to wrap your fetch calls in separate functions. 

```js
const getUser = (userId) => {
  const baseUrl = 'https://reqres.in/api/users/'
  return fetch(`${baseUrl}/${userId}`)
                 .then((res) => res.json );
}

useEffect(() => {
  getUser(props.userId).then( // set state)
})
```